libr/core/rtr.c

```diff
@@ -386,7 +390,12 @@ static void activateDieTime (RCore *core) {
 	int dt = r_config_get_i (core->config, "http.dietime");
 	if (dt > 0) {
 #if __UNIX__
-		signal (SIGALRM, dietime);
+		struct sigaction sigact;
+		sigact.sa_handler = dietime;
```

Comment:
What about adding an r_sys_signal() api instead of having to change all that code with ifdefs for windows/unix?. this will be cleaner